### ISSUE TITLE
SSX Free does not show when server is offline when it is offline

### ISSUE THREAD

**Issue by Blocksnmore** [ServerSelectorX/ServerSelectorX#153]
I am using the legacy version (the big text) of ssx and I have an issue where the server will never be shown as offline when it is offline for me. here is a part of my config. I get no errors in console

```
menu:

  # Item in 4th slot (use 0 for first item). Add as many items as you want by copying this section and giving it a new number.
  3:

    # https://github.com/ServerSelectorX/ServerSelectorX/wiki/Free-%7C-Server-Pinging
    ping-server: true
    ip: not.a.real.ip
    port: 00006
    
    # https://github.com/ServerSelectorX/ServerSelectorX/wiki/Actions
    action: 'cmd:bungeeplayer move minigamehub'

    # Change item count based on number of players online. Only works if server pinging is set up. 
    # If there are more than 64 players online, the item count will be 1. Change how this works in config.yml.
    change-item-count: true
    
    # This value is ignored when change-item-count is turned on.
    item-count: 1

    # Online is used when ping-server is turned off, or ping-server is turned on and the server is online.
    online:
      item: WOOD
      data: 0
      name: '&aKitPvP Server'
      lore:
      # Placeholders not working? https://github.com/ServerSelectorX/ServerSelectorX/wiki/FAQ#placeholders-are-not-working
       - 'There are {online} out of {max} players online.'
       - 'Message of the day: {motd}'
      
    # Offline is used when ping-server is turned on and the server is offline.
    offline:
      item: WOOL
      data: 0
      name: '&cKitPvP Server'
      lore:
       - Ipsum lorem
       - Amet sit dolor
       - The server is offline
```

it always shows the online message no matter what. I am using the default config with only a few things changed to make sure it is as accurate as possible to the default config

**Comment by Blocksnmore** [comment]
Github messed up the formatting a bit but it is still accurate

**Comment by Derkades** [comment]
Note that it can take a while to update in the free version, especially if you have many servers. It can from a few seconds up to a few minutes.

**Comment by Blocksnmore** [comment]
The server it was pinging was offline before the server even started

**Comment by Derkades** [comment]
The IP or port must not be correct

**Comment by Blocksnmore** [comment]
I have checked my config and the Ip and it is proper

**Comment by Derkades** [comment]
Can you try the newly released version?

**Comment by EpicnessTwo** [comment]
I'm also having this exact issue. I've tried both the internal and external ping as well as the main server port and the query port. querying is enabled on the servers.

I do however run all of my servers on the IP of 127.0.0.1 however as they're all on the same machine.

Despite them stating they're offline, I can still click on them and join just fine so I might have to accept that I'll just have to use the offline message and not have a player count.

How hard would it be to add a debug mode so that we can see when each query ping is sent out, the result of said ping and what the plugin is doing about it? It might make it a lot easier to diagnose where the issue could be...

**Comment by Derkades** [comment]
That's a different issue, this issue is about servers showing as online while they are offline. You can't use localhost or internal ip addresses: https://github.com/ServerSelectorX/ServerSelectorX/wiki/Free-%7C-Server-Pinging

**Comment by EpicnessTwo** [comment]
Ah sorry about that... I must have misread the original issue.

**Comment by Blocksnmore** [comment]
Now when I try to open the menu with the latest version it says "missing item option for number 5" when it is set, this might be a config issue so I will try remaking the menu

**Comment by Blocksnmore** [comment]
with the default config it shows the same error but for item -1

**Comment by Derkades** [comment]
Currently the legacy versions has some issues. You can fix the missing item -1 option by changing `online` to `offline` for that item, but there are some other issues as well

**Comment by Derkades** [comment]
everything is fixed in 3.5.1
